client/src/components/Dialogs/CartModal.js

```diff
@@ -0,0 +1,264 @@
+                size="small"
+                aria-label="a dense table"
+              >
+                <TableHead>
+                  <TableRow>
+                    <TableCell></TableCell>
+                    <TableCell align="center">Menu</TableCell>
+
+                    <Hidden smDown>
+                      <TableCell align="left">Qty</TableCell>
+                    </Hidden>
+                    <Hidden smDown>
+                      <TableCell align="center">Price</TableCell>
+                    </Hidden>
+
+                    <TableCell align="right"></TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {cart.map(({ qty, recipe }) => (
+                    <TableRow key={recipe._id}>
+                      <TableCell component="th" scope="item">
+                        <Avatar
+                          alt="Remy Sharp"
+                          src={recipe.recipePictureUrl || plate}
+                        />
+                      </TableCell>
+                      <TableCell align="justify">{recipe.name}</TableCell>
+                      <Hidden smDown>
+                        <TableCell align="justify" nowrap="nowrap">
+                          <span>{qty}</span>{" "}
+                          <ButtonGroup
+                            disableElevation
+                            variant="contained"
+                            className={classes.btnGroup}
+                          >
+                            <Button
+                              onClick={() =>
+                                qty > 1
+                                  ? decreaseCount(dispatch, {
+                                      id: recipe._id,
+                                      qty: qty - 1,
+                                      chef: recipe.user,
+                                    })
+                                  : removeFromCart(dispatch, recipe._id)
+                              }
+                            >
+                              -
+                            </Button>
+                            <Button
+                              onClick={() =>
+                                increaseCount(dispatch, {
+                                  id: recipe._id,
+                                  qty: qty + 1,
+                                  chef: recipe.user,
+                                })
+                              }
+                            >
+                              +
+                            </Button>
+                          </ButtonGroup>
+                        </TableCell>
+                      </Hidden>
+                      <Hidden smDown>
+                        <TableCell align="right">{`${
+                          recipe.price
+                        }x${qty}=${addDecimals(
+                          qty * recipe.price
+                        )}`}</TableCell>
```

Comment:
I think you could increase the spacing of this both in the UI and in the code for readability of both